WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift

```diff
@@ -98,8 +98,8 @@ class NotificationSettingDetailsViewController: UITableViewController {
 
     private func setupTableView() {
         // Register the cells
-        tableView.register(SwitchTableViewCell.self, forCellReuseIdentifier: Row.Kind.Setting.rawValue)
-        tableView.register(WPTableViewCell.self, forCellReuseIdentifier: Row.Kind.Text.rawValue)
+        tableView.register(SwitchTableViewCell.self, forCellReuseIdentifier: CellKind.Setting.rawValue)
+        tableView.register(WPTableViewCellValue1.self, forCellReuseIdentifier: CellKind.Text.rawValue)
```

Comment:
not really a great naming, but it's pre-existing..

Reply:
Yeah, these things are everywhere!

Reply:
Yeah, there a whole bunch of pre-existing cell types like this